dockers/skylab/HCA_post_processing/optimus_HCA_loom_merge.py

```diff
@@ -85,11 +87,12 @@ def main():
                 # filter out cells with low counts n_molecules > 1
                 UMIs = ds.ca['n_molecules']
                 cells = np.where(UMIs >= 100)[0]
-                for (ix, selection, view) in ds.scan(items=cells, axis=1, key="gene_names"):
+                for (ix, selection, view) in ds.scan(items=cells, axis=1): 
+                    view.ca['cell_names'] = view.ca['cell_names'] + "-" + str(i)
```

Comment:
did you remove the call that does this in the original file?

Reply:
discussed and already removed.